mrf/era-1/basic-syntax-#9.js

```diff
@@ -0,0 +1,186 @@
+// add number a to b
+function add(a, b) {
+  return a + b;
+}
+
+//! test add func
+console.log("add 1 to 1:", add(1, 1));
```

Comment:
This is not part of the task, but instead of logging, a better approach is to _`assert`_ the result of calling function is exactly what you expect, e.g.:
```suggestion
console.assert(add(1, 1) === 2, { a: 1, b: 1, function: add.name });
```
Here, you say "I'm asserting `add(1, 1)` is `2`. If not, write the provided object to the console."

Note that logging is ok and part of many projects. I'm just trying to provide you alternative approaches which may or may not be good based on the context.

Reply:
got it :) 
🌾🌾🌾